projectDocs/dev/developerGuide/developerGuide.md

```diff
@@ -1077,6 +1083,38 @@ To allow plugins in your add-on to access gettext message information via calls
 This function cannot be called in modules that do not belong to an add-on, e.g. in a scratchpad subdirectory.
 For more information about gettext and NVDA translation in general, please read the [Translating NVDA page](https://github.com/nvaccess/nvda/blob/master/projectDocs/translating/readme.md)
 
+#### SPeech symbol dictionaries {#AddonSymbolDictionaries}
+
+Starting with NVDA 2024.4, you can provide custom speech symbol dictionaries in add-ons to improve symbol pronunciation.
```

Comment:
I think we should avoid making these docs temporal, instead the information should match what's in accompanied released code implicitly. We rarely qualify the release when documenting other API features in this document.

```suggestion
You can provide custom speech symbol dictionaries in add-ons to improve symbol pronunciation.
```